.github/workflows/build.yml

```diff
@@ -151,7 +141,7 @@ jobs:
           ./configure --no-tests ${{ env.CONFIG_OPTIONS }}
           tools/scripts/show_build_config.pl
           . setenv.sh
-          make -j3 ${{ env.BUILD_TARGETS }}
+          make -j4 ${{ env.BUILD_TARGETS }}
```

Comment:
I've seen on projects where something like that can fail because a VM doesn't have enough memory to handle that many compiles at once.

Reply:
That's a good point, but the only time I think we normally run into that for OpenDDS is when we're compiling TypeSupport for very large test IDL files. Since this is building OpenDDS without tests I doubt it would ever be an issue here.